PEP 752: Updates including co-authoring of the PEP (#4292)
python/peps: peps/pep-0752.rst

__ https://github.com/pypi/warehouse/blob/8615326918a180eb2652753743eac8e74f96a90b/warehouse/migrations/versions/d18d443f89f0_ultranormalize_name_function.py#L29-L42

Another problem that namespacing would solve is the issue of choosing new names
for packages following the agreed patterns of naming. Often (this is the case
for ``Apache Airflow`` for example), there are public discussions that precede
the decision to create a new package. The decision is based on the agreed
name and follow the pattern of the existing packages. If more package names are

> considered during the discussion, all the names have to be reserved via ``PyPI``

Reviewer: Does this PyPI interface for reserving names already exist?
```suggestion
considered during the discussion, all the names have to be reserved via a PyPI
```

Reviewer: (Unresolving this because I had a question about the PyPI interface.)

Author: Yep it exists

<img width="766" alt="Screenshot 2025-03-05 at 22 45 20" src="https://github.com/user-attachments/assets/c6df0473-2cad-4118-94ff-16dd1e72ca46" />

interface before the discussion is public, otherwise the names can be taken by

> other users. This is a problem that actually happened in the past as explained
> in the associated discussion.

Reviewer: Can we link to the associated discussion?
```suggestion
other users. This has happened in the past as explained
in the associated discussion.
```

Author: I added the discussion link
